apps/backend-functions/src/notification.ts

```diff
@@ -678,9 +678,12 @@ async function sendContractStatusChangedConfirmation(recipient: User, notificati
     contract, title, app, isRecipientBuyer, toUser, recipientOrg, counterOfferSenderOrg
   } = await getNegotiationUpdatedEmailData(recipient, notification);
 
+  let pageURL = `${appUrl.content}/c/o/dashboard/sales/${contract.id}/view`;
+  if(isRecipientBuyer) pageURL = `${appUrl.content}/c/o/marketplace/offer/${contract.offerId}/${contract.id}`;
+
   const data = {
-    user: toUser, baseUrl: appUrl.content, org: recipientOrg,
-    contract, title, isRecipientBuyer
+    user: toUser, org: recipientOrg,
+    contract, title, pageURL, app:{name:appName.catalog}
```

Comment:
Give the object some air (a.k.a. space) ^^
```suggestion
  const data = {
    user: toUser,
    org: recipientOrg,
    contract,
    title,
    pageURL,
    app: { name: appName.catalog }
  };
```